tools/Dockerfile

```diff
@@ -93,12 +85,7 @@ RUN cp -p LICENSE /usr/share/licenses/eksctl-anywhere && \
       --spdx-data /usr/libexec/tools/spdx-data \
       --out-dir /usr/share/licenses/eksctl-anywhere/vendor \
       go-vendor ./vendor
-RUN curl -L "${EKSA_BINARY_URL}" \
-      -o "eksctl_anywhere_${EKSA_VERSION}_${EKSA_RELEASE_NUMBER}_${GOOS}_amd64.tar.gz" && \
-    grep "eksctl_anywhere_${EKSA_VERSION}_${EKSA_RELEASE_NUMBER}_${GOOS}_amd64.tar.gz" \
-      /hashes/eksctl-anywhere | sha512sum --check - && \
```

Comment:
We would no longer need the associated hash in https://github.com/bottlerocket-os/bottlerocket-test-system/blob/develop/tools/hashes/eksctl-anywhere

Reply:
We could still do hash validations. If the source was modified, an additional parameter can be given with the new expected hash.